boot/menhir/parser.mli

```diff
@@ -83,6 +83,7 @@ type token =
   | HASH
   | GREATERRBRACKET
   | GREATERRBRACE
+  | GREATERDOT
```

Comment:
Don't we want to abstract the token name since we disagree on the concrete token?
Maybe
- `META_BRACKET_CLOSE`
- `META_BRACKET_OPEN`
- `META_ESCAPE`
?

Reply:
I thought about this, but then I decided that if the goal is to make Oleg's life easier, maybe forcing non-essential renames was not the best approach?

Reply:
(I'm happy to do the rename if you thought about this even slightly more and still believe that it's the right approach.)

Reply:
This aligns with a suggestion from Oleg (in an email thread), so I did the rename. I used METAOCAML_ instead of META_ as a prefix.